settings.gradle

```diff
@@ -28,3 +28,4 @@ include 'examples:banananation'
 include 'examples:foo-missionmodel'
 include 'examples:config-with-defaults'
 include 'examples:config-without-defaults'
+include 'aerielander'
```

Comment:
aerielander has not been cleared yet (to the best of my knowledge) so building would fail for someone who would not have it in their directory